src/library/ListItem/Labels/Identity.tsx

```diff
@@ -39,7 +33,7 @@ export const Identity = ({ address }: IdentityProps) => {
     >
       <PolkadotIcon dark={mode === 'dark'} nocopy address={address} size={24} />
       <div className="inner">
-        {identitiesSynced && supersSynced && display !== null ? (
```

Comment:
If no validators have a supers name, the display stops rendering, even if they have an identity name. This is because the length `supersSynced` is being checked. There's no need to check if it’s loading, as the name won’t display regardless, since there’s no loading icon

Reply:
The issue was that the validator name was not displaying.